Helper/Data.php

```diff
@@ -49,17 +56,40 @@ protected function getScopeSetting($path){
 
         if (isset($storeId)) {
             $scope = \Magento\Store\Model\ScopeInterface::SCOPE_STORE;
-            $value = $storeId;
-            return $this->_scopeConfig->getValue($path, $scope, $value);
+            $scopeCode = $storeId;
```

Comment:
why don't we just use `$storeId`?

Reply:
good question, this change was part of my variable name cleanup efforts and I had left the original structure intact. you could make an argument that this is a bit more readable, but I have no problems with changing it

Reply:
i mean maybe just rename the original $storeId variable to $scopedStoreCode?  then just use that?